ci/Jenkinsfile.tests

```diff
@@ -52,16 +52,24 @@ pipeline {
             """
           }
         }
-        stage('Tests') {
+        stage('Unit Tests') {
           steps {
             sh """#!/bin/bash
               set -eo pipefail
-              make test 2>&1 | tee -a ${LOG_FILE}
+              make unit-test 2>&1 | tee -a ${LOG_FILE}
             """
           }
         }
       }
     }
+    stage('Integration Tests') {
```

Comment:
Can those not run in parallel?

Reply:
I mentioned this in the PR description:

> Integration Tests run later because running them at the same time as Unit Tests caused errors (see https://ci.status.im/blue/organizations/jenkins/status-mobile%2Fprs%2Ftests/detail/PR-18304/2/pipeline)

Originally, I wanted to run them all in parallel. I ran the pipeline twice and faced that error and honestly didn't care about solving the problem, not now at least. It's a good question nonetheless.

The same question can be made about the component tests, they are currently not running in parallel with the other checks and I don't know why.

Reply:
@jakubgs, I would like to keep the integration tests separate from unit test because I don't want to dedicate even more time to this PR. In the future somebody can figure out if & how we can run integration and component tests in parallel with unit tests and linting.

Reply:
That makes sense.